1-exercises/B-object-arrays/exercise-1.js

```diff
@@ -13,8 +13,12 @@ let people = [
   { name: "Elise", age: 35 },
 ];
 
-let peopleNames =
-  /* todo */
+let peopleNames = people.map(findName);
```

Comment:
nice use of `map` - this could be ` const` instead of `let` as you're not redefining the variable